### PR TITLE
bugfix/a11y-series-description

### LINKED ISSUE #20656: Default series description not describing all series in a Stock chart.
#### Expected behaviour
I expect all three series to have a default series description.

#### Actual behaviour
`aria-label` is missing for the first two series. When navigating with screen reader, skips first two series and goes to last series and describes "Navigator 1, series 3 of 3.... ". What seems to be helping is clicking one of the range selector buttons, and then clicking "All"-button. Description is then working for the other two series. Could potentially be something with async data-loading and not related to Stock since it is working after clicking one of the range selector buttons? 

#### Live demo with steps to reproduce
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/demo/stock-tools-gui

#### Product version
Highcharts Stock

#### Affected browser(s)
Chrome, Safari with VoiceOver from MacOS Sonoma 14.2.1

### PR DESCRIPTION
**Pull request by pawellysy**
Fixed #20656, series with manipulated points were missing a11y description on initial load.

### CODE REVIEW

**Review by pawelfus — APPROVED** [review]
Thanks!

**Review by kamil-musialowski — APPROVED** [review]
Good one!

File: samples/unit-tests/accessibility/descriptions-added/demo.js [hunk 1]
```diff
@@ -154,4 +154,25 @@ QUnit.test('Proxy region', function (assert) {
...
+                    enabled: true,
+                    forced: true
+                }
+            }
+        ]
+    });
+
+    assert.ok(
+        chart.series[0].group.element.getAttribute('aria-label'),
+        'Group element has ariaLabel.'
```
Comment (DJTechnoo) [inline samples/unit-tests/accessibility/descriptions-added/demo.js:176]: Perhaps change to "Group element should have aria label"

**Review by DJTechnoo — APPROVED** [review]
Approving, but have one small comment!

**Review by bm64 — APPROVED** [review]

**Review by marvin19 — APPROVED** [review]
Tested with VoiceOver 👍

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**